### PR TITLE
Update pFL-Bench scripts and README for the appendix-included version

### PR DESCRIPTION
**Pull request by yxdyc**
- update the pFL-Bench scripts according to the paper versioned by appendix-added

- make the pFL-Bench directory name consistent with the paper link

### CODE REVIEW

File: benchmark/pFL-Bench/README.md [hunk 1]
```diff
@@ -36,6 +36,13 @@ You can find the full scripts from the another [branch](https://github.com/aliba
 We put all the config yaml file in the directory `benchmark/pfl_bench/yaml_best_runs`.
 To reproduce the experiments with searched best configurations, you can run the experiment as the following example:
 ```
+## Pre-step-1: If you do not use our docker, please install Federated Scope according to the pre-steps
+# git clone https://github.com/alibaba/FederatedScope.git
```
Comment (DavdGao) [inline benchmark/pFL-Bench/README.md:40]: should not be commented out

File: benchmark/pFL-Bench/README.md [hunk 2]
```diff
@@ -36,6 +36,13 @@ You can find the full scripts from the another [branch](https://github.com/aliba
...
+## Pre-step-2: follow https://github.com/alibaba/FederatedScope/blob/master/README.md to build the running environment
+## Pre-step-3: switch to the branch `feature/pFL_bench` if you want to run the codes with the version for paper's experiments.
+# git switch Feature/pfl_bench
```
Comment (DavdGao) [inline benchmark/pFL-Bench/README.md:43]: the same as above

**Review by DavdGao — COMMENTED** [review]
please see the inline comments

File: benchmark/pFL-Bench/README.md (hunk 1 above)
Comment (yxdyc) [reply benchmark/pFL-Bench/README.md]: Modified accordingly.

File: benchmark/pFL-Bench/README.md (hunk 2 above)
Comment (yxdyc) [reply benchmark/pFL-Bench/README.md]: Modified accordingly.

**Review by xieyxclack — APPROVED** [review]
LGTM

**Pull request merged**